dart/lib/src/sentry.dart

```diff
@@ -357,6 +360,22 @@ class Sentry {
   /// Gets the current active transaction or span bound to the scope.
   static ISentrySpan? getSpan() => _hub.getSpan();
 
+  static Future<void> addFeatureFlag(String name, bool value) async {
+    final featureFlagsIntegration = currentHub.options.integrations
+        .whereType<FeatureFlagsIntegration>()
+        .firstOrNull;
+
+    if (featureFlagsIntegration == null) {
+      currentHub.options.logger(
+        SentryLevel.debug,
```

Comment:
We are going to change the default log level to warning soon, maybe this should be a log warning?